openmetadata-ui/src/main/resources/ui/src/pages/UserPage/UserPage.component.tsx

```diff
@@ -179,7 +180,7 @@ const UserPage = () => {
     fetchUserData();
   }, [username]);
 
-  if (isLoading) {
+  if (userData?.name !== username || isLoading) {
```

Comment:
Not sure on this condition, Isn't data fetching should be handling loading.
I think we can remove this condition

Reply:
There was 1 additional feeds(normal) call happening when then user profile was changed along with the feeds call happening for that particular user.
This additional condition was added to prevent that additional feeds api call.